parts/linux/cloud-init/artifacts/ci-syslog-watcher.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+[ ! -f "/var/run/mdsd/update.status" ] && exit 0
+status=$(cat /var/run/mdsd/update.status)
+
+if [[ "$status" == "add" ]]; then
+        echo "Status changed to $status."
+        cat >/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf << 'EOL'
+$ModLoad omuxsock 
+$OMUxSockSocket /var/run/mdsd/default_syslog.socket 
+template(name="MDSD_RSYSLOG_TraditionalForwardFormat" type="string" string="<%PRI%>%TIMESTAMP% %HOSTNAME% %syslogtag%%msg:::sp-if-no-1st-sp%%msg%")
+$OMUxSockDefaultTemplate MDSD_RSYSLOG_TraditionalForwardFormat
+*.* :omuxsock:
+EOL
+elif [[ "$status" == "remove" ]]; then
+        echo "Status changed to $status."
+        [ -f "/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf" ] && rm /etc/rsyslog.d/60-rsyslog-forward-mdsd.conf
```

Comment:
will this path "/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf" conflicts with other MDSD running on the AKS node?

Reply:
currently there is not other scenario where mdsd is running on the node. however even if it was, upon renaming the config file and the socket there will not be any conflicts

Reply:
won't 1p often have mdsd (I guess that would require them having container monitoring enabled too)

Reply:
we will be starting mdsd at a custom location to prevent any conflicts